doc/content/verification_and_validation/ver-1dd.md

```diff
@@ -0,0 +1,49 @@
+# ver-1dd
+
+# Permeation Problem without Trapping
+
+## Test Description
```

Comment:
The test description should list the values of the model parameters and case configurations, like the thickness of the plate, the diffusivity, etc. 
It is OK to say: 
`we use the same values as in [link to ver-1d],`
 but you need to make sure there is no confusion for the user.